### PR TITLE
fix(COD-1136) don't upload policies to CDS

### PR DESCRIPTION
**Pull request by satellite15**
https://lacework.atlassian.net/browse/COD-1130

disabling upload of policies to CDS

### CODE REVIEW

**Comment by satellite15** [comment]
@hemanthgk10 anything else we should disable?

**Comment by hemanthgk10** [comment]
> @hemanthgk10 anything else we should disable?

The only other thing I can think of is the inventory command that can also upload data to CDS. As we are planning everything to be in CDS, I don't see a requirement for this but one thing that comes to my mind is maybe we just need fix the tag.. initially I was sending the [module name as tag](https://github.com/lacework/rainbow/pull/12452/files) then realized we should just send as iac.

Now I feel, we should split and fix the tags correctly as part of replatforming.

* IAC tag - for all the assessment related data.
* Module as tag - for all the non assessment related data.

File: pkg/api/client.go [hunk 1]
```diff
@@ -325,7 +332,7 @@ func uploadResultsToCDS(c *Client, filesToUpload []string) error {
 			// log.Errorf("{warning:Unable to upload results %s\n}", err)
 			return err
 		}
-		log.Infof("Successfully uploaded to CDS: {info:%s}", guid)
+		log.Debugf("Successfully uploaded to CDS: {info:%s}", guid)
```
Comment (hemanthgk10) [inline pkg/api/client.go:335]: I understand why you are changing this, but there is one issue so far. There is no way of tracking if we don't no the upload guid unfortunately till we get EVAL_GUID that is constant across to track.

**Review by hemanthgk10 — APPROVED** [review]
LGTM, except that small concern I have.

File: pkg/api/client.go (hunk 1 above)
Comment (satellite15) [reply pkg/api/client.go]: @hemanthgk10 I have kept this as info but made it a little more user friendly
```
		log.Infof("Assessment upload reference: {info:%s}", guid)
```

**Pull request merged**